src/components/FieldLabel/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FieldLabel = (props) => {
+  let requireText = '';
+  if (props.required) {
+    requireText = <span className="text-danger" aria-hidden> * Required</span>;
+  } else if (props.requiredForSubmit) {
+    requireText = <span className="text-info" aria-hidden> * Required for submission</span>;
+  }
+
+  return (
+    <div className={props.className}>
+      <strong>{props.text}:</strong>
+      {requireText}
+    </div>
+  );
+};
+
+FieldLabel.defaultProps = {
+  className: '',
+  required: false,
+  requiredForSubmit: false,
```

Comment:
You don't have `requiredForSave` in the `propTypes`, but you do use it somewhere else. Maybe a mixup in language?

Reply:
Yes, thank you. I renamed a property last minute. See! Reviews are good after all.